doc/classes/Rect2.xml

```diff
@@ -211,7 +244,7 @@
 			<return type="bool" />
 			<param index="0" name="right" type="Rect2" />
 			<description>
-				Returns [code]true[/code] if the rectangles are exactly equal.
+				Returns [code]true[/code] if the [member position] and [member size] of the rectangles are exactly equal.
```

Comment:
```suggestion
				Returns [code]true[/code] if both [member position] and [member size] of the rectangles are exactly equal, respectively.
```
(I don't think "exactly" is needed but oh well)

Reply:
I wanted to specify here because there's 4 very precise floating-point numbers that have to be equal to return `true`. I suppose the adjective is not really necessary for Rect2i, however.

Reply:
Yeah, I get the reasoning. I mean "equal" and "exactly equal" are the same thing. :upside_down_face: But if it's clearer then sure.
(and I skipped / haven't looked at the Rect2i part)